src/modules/LanUdevAuto.ycp

```diff
@@ -210,7 +211,7 @@ global define boolean Write()
 
         string dev = rule["rule"]:"" == "KERNELS" ? FindBy(MatchBusId, rule["value"]:"") : FindBy(MatchMac, rule["value"]:"");
 
-        if( !IsEmpty(dev)))
```

Comment:
This fix should go in a commit preceding this one, saying that it is a fix that *did* make it to the PTF which the customer actually tested.